$MirOS: ports/devel/autoconf/2.62/patches/patch-lib_autoconf_programs_m4,v 1.1 2009/11/22 00:16:23 tg Exp $
--- lib/autoconf/programs.m4.orig	Sat Apr  5 23:04:25 2008
+++ lib/autoconf/programs.m4	Sat Jan 28 14:14:35 2017
@@ -617,7 +617,7 @@ rm -rf conftest.one conftest.two conftes
     # value for INSTALL within a source directory, because that will
     # break other packages using the cache if that directory is
     # removed, or if the value is a relative name.
-    INSTALL=$ac_install_sh
+    INSTALL="$SHELL $ac_install_sh"
   fi
 fi
 dnl Do special magic for INSTALL instead of AC_SUBST, to get
@@ -917,8 +917,8 @@ AN_PROGRAM([bison], [AC_PROG_YACC])
 AC_DEFUN([AC_PROG_YACC],
 [AC_CHECK_PROGS(YACC, 'bison -y' byacc, yacc)dnl
 AC_ARG_VAR(YACC,
-[The `Yet Another C Compiler' implementation to use.  Defaults to the first
-program found out of: `bison -y', `byacc', `yacc'.])dnl
+[The `Yet Another Compiler Compiler' implementation to use.  Defaults to
+the first program found out of: `bison -y', `byacc', `yacc'.])dnl
 AC_ARG_VAR(YFLAGS,
 [The list of arguments that will be passed by default to $YACC.  This script
 will default YFLAGS to the empty string to avoid a default value of `-d' given
